.github/workflows/ci.yml

```diff
@@ -15,6 +15,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
Nice, today I learned

Reply:
Why do we need it? Please add a comment to the yml file